gui/tools/arc_summary.py

```diff
@@ -838,9 +840,13 @@ def get_l2arc_summary(Kstat):
         output["spa_mismatch"] = fHits(l2_write_spa_mismatch)
 
         output["l2_arc_size"] = {}
-        output["l2_arc_size"]["adative"] = fBytes(l2_size)
+        output["l2_arc_size"]["adaptive"] = fBytes(l2_size)
+        output["l2_arc_size"]["actual"] = {
+            'per': fPerc(l2_asize, l2_size),
+            'num': fBytes(l2_asize),
+        }
         output["l2_arc_size"]["head_size"] = {
-            'per': fPerc(l2_hdr_size, l2_size),
+            'per': fPerc(l2_hrd_size, l2_size),
```

Comment:
This is the typo travis is complaining about

Reply:
Sorry about that!